flexmeasures/utils/config_utils.py

```diff
@@ -72,11 +72,27 @@ def check_app_env(env: str | None):
 def read_config(app: Flask, custom_path_to_config: str | None):
     """Read configuration from various expected sources, complain if not setup correctly."""
 
-    check_app_env(app.env)
+    flexmeasures_env = "production"
+    if app.testing:
+        flexmeasures_env = "testing"
+    elif os.getenv("FLEXMEASURES_ENV", None):
+        flexmeasures_env = os.getenv("FLEXMEASURES_ENV", None)
+    elif os.getenv("FLASK_ENV", None):
+        flexmeasures_env = os.getenv("FLASK_ENV", None)
+        import warnings
+
+        warnings.warn(
+            "'FLASK_ENV' is deprecated and replaced by FLEXMEASURES_ENV"
+            " Change FLASK_ENV to FLEXMEASURES_ENV in the environment variables",
+            DeprecationWarning,
+            stacklevel=2,
+        )
+
+    check_app_env(flexmeasures_env)
```

Comment:
Within this function, our error message says "Flask(flexmeasures) environment ...".

We could say "FLEXMEASURES_ENV (or the `env` parameter)" now right?

Reply:
updated the message